bedevere/prtype.py

```diff
@@ -17,13 +17,15 @@ class Labels(enum.Enum):
     performance = "performance"
     type_security = f"{TYPE_LABEL_PREFIX}-security"
     tests = "tests"
+    skip_news = "skip news"
 
 
-async def add_label(gh, issue, label):
+async def add_label(gh, issue, labels):
     """Apply this type label if there aren't any type labels on the PR."""
     if any(label.startswith("type") for label in util.labels(issue)):
         return
```

Comment:
```suggestion
    """Add the specified labels to the PR."""
```
I think this part could be removed, since it's based on the assumption that all the labels passed are `type-*` labels and the fact that the PR can only have one `type-*` label, even though now it's being used for other labels too and `docs`/`tests`/`performances` are no longer `type-*` labels.

Reply:
I do not completely follow you. Isn't this a check that makes sure we are not overwriting the current label types if there are already specific `type-*` labels?

Reply:
Oh never mind, didn't see you other suggestion.